src/main/java/randoop/sequence/VariableRenamer.java

```diff
@@ -22,32 +25,37 @@ public VariableRenamer(Sequence sequence) {
    * Heuristically transforms variables to better names based on its type name. Here are some
    * examples:
    *
+   * <p>
```

Comment:
This shouldn't be here.  I'm surprised it can be; didn't the formatter remove it?